ttl/hbp_taxonomy_ontology.ttl

```diff
@@ -30,6 +30,7 @@ obo:NCBITaxon_10090 rdf:type owl:Class ;
                        rdfs:label "Mus musculus"@en ;
                        rdfs:subClassOf obo:NCBITaxon_9989 ;
                        HBP_PROP:has_rank HBP_TAX:9999996 ;
+                       HBP_PROP:has_brain_atlas_ontology HBP_MBA;
```

Comment:
Is the property HBP_PROP:has_brain_atlas_ontology defined in this file or in /ttl/hbp_properties.ttl ?
If not, it should be defined in /ttl/hbp_properties.ttl.